src/components/tooltip/tooltip-theme.scss

```diff
@@ -1,6 +1,6 @@
-md-tooltip.md-THEME_NAME-theme {
-  color: '{{background-700-contrast}}';
-  .md-content {
-    background-color: '{{background-700}}';
+.md-tooltip.md-THEME_NAME-theme {
+  color: '{{background-A100}}';
```

Comment:
The colors shouldn't be changed here. It seems you didn't rebase properly here after this PR:

https://github.com/angular/material/pull/9207